index.rst

```diff
@@ -77,19 +85,39 @@ categorized as:
 Tag Fragments
 -------------
 
-In general, tags can be postfixed with an optional Cycle number or a
-``rest`` set of one-or-more underscore-separated fields.  The Cycle is
-currently unique to T&S RSP instances, and specifies the version of XML
-used in this image.  It will have the form ``c<digits>.<digits>`` where
-the first group of digits is the cycle number and the second group of
-digits is the build iteration for that cycle, e.g. ``c0019.001``.
+Primary
+^^^^^^^
+The primary version of a tag is derived from (and in most cases identical to) the EUPS tag of the underlying DM stack included in the image.
+It is always the first component of the tag.
+
+RSP Build Counter
+^^^^^^^^^^^^^^^^^
+
+Tags may include an identifier specifying an RSP image build count. 
+In general, any run of the GitHub Action that builds a sciplat-lab image will increment this counter.
+It is currently in the 2000s, and we do not anticipate it exceeding 10,000 during the lifetime of the project.
+We anticipate only including this counter in Release, Release Candidate, and Experimental builds derived from one of the former two categories, although it will continue to increment with each build regardless of type.
+
+If specified, this goes directly after the primary component.
+It takes the form ``rspX`` where ``X`` is an integer and is separated from the primary tag with an underscore.
+
+Tags can additionally be postfixed with an optional Cycle number or a ``rest`` set of one-or-more underscore-separated fields.
```

Comment:
I would delete this since that's already covered by the following sections and those bits are not part of the RSP build counter.